README.md

```diff
@@ -64,3 +64,13 @@ This project uses pytest. To run all the tests use:
 ```bash
 $ poetry run pytest tests
 ```
+
+## Ansible
+
+If changing or adding new files for ansible you will want to load them into the control node by running the following in the root of this project.
+
+```bash
+$ sh ansible/copy_to_vm.sh
+```
+
+This will copy all files from `ansible/vm_files/` into `/home/ec2-user` on the control node
```

Comment:
Minor, but it would be nice to also mention how to run the ansible commands on the control node here